src/foundry/foundry.js/avClient.d.ts

```diff
@@ -8,10 +8,36 @@ declare abstract class AVClient {
    */
   constructor(master: AVMaster, settings: AVSettings);
 
+  /**
+   * The master orchestration instance
+   */
   master: AVMaster;
 
+  /**
+   * The active audio/video settings being used
+   */
   settings: AVSettings;
 
+  /**
+   * Is audio broadcasting push-to-talk enabled?
+   */
+  get isVoicePTT(): boolean;
+
+  /**
+   * Is audio broadcasting always enabled?
+   */
+  get isVoiceAlways(): boolean;
+
+  /**
+   * Is audio broadcasting voice-activation enabled?
+   */
+  get isVoiceActivated(): boolean;
+
+  /**
+   * Is the current user muted?
+   */
+  get isMuted(): false;
```

Comment:
I believe this should be `boolean`, not `false`.

Reply:
done